hadoop-hdds/server-scm/src/main/resources/webapps/scm/scm.js

```diff
@@ -142,6 +168,46 @@
+                    ctrl.scmpipelinemanager.PipelineInfo.forEach(({key, value}) => {
+                        if(key == "CLOSED") {
+                            $scope.statistics.nodes.data.pipeline.closed = value;
+                        } else if(key == "ALLOCATED") {
+                            $scope.statistics.nodes.data.pipeline.allocated = value;
+                        } else if(key == "OPEN") {
+                            $scope.statistics.nodes.data.pipeline.open = value;
+                        } else if(key == "DORMANT") {
+                            $scope.statistics.nodes.data.pipeline.dormant = value;
+                        }
+                    });
+                });
+
+            $http.get("jmx?qry=Hadoop:service=StorageContainerManager,name=ReplicationManagerMetrics")
+                .then(function (result) {
+                    const URLScheme = location.protocol.replace(":" , "");
+                    ctrl.scmcontainermanager = result.data.beans[0];
+                    $scope.statistics.nodes.data.container.open = ctrl.scmcontainermanager.OpenContainers;
+                    $scope.statistics.nodes.data.container.closing = ctrl.scmcontainermanager.ClosingContainers;
+                    $scope.statistics.nodes.data.container.quasi_closed = ctrl.scmcontainermanager.QuasiClosedContainers;
+                    $scope.statistics.nodes.data.container.closed = ctrl.scmcontainermanager.ClosedContainers;
+                    $scope.statistics.nodes.data.container.deleting = ctrl.scmcontainermanager.DeletingContainers;
+                    $scope.statistics.nodes.data.container.deleted = ctrl.scmcontainermanager.DeletedContainers;
+                    $scope.statistics.nodes.data.container.recovering = ctrl.scmcontainermanager.RecoveringContainers;
+                    $scope.statistics.nodes.data.container.under_replicated = ctrl.scmcontainermanager.UnderReplicatedContainers;
+                    $scope.statistics.nodes.data.container.mis_replicated = ctrl.scmcontainermanager.MisReplicatedContainers;
+                    $scope.statistics.nodes.data.container.over_replicated = ctrl.scmcontainermanager.OverReplicatedContainers;
+                    $scope.statistics.nodes.data.container.missing = ctrl.scmcontainermanager.MissingContainers;
+                    $scope.statistics.nodes.data.container.unhealthy = ctrl.scmcontainermanager.UnhealthyContainers;
+                    $scope.statistics.nodes.data.container.empty = ctrl.scmcontainermanager.EmptyContainers;
+                    $scope.statistics.nodes.data.container.open_unhealthy = ctrl.scmcontainermanager.OpenUnhealthyContainers;
+                    $scope.statistics.nodes.data.container.quasi_closed_stuck = ctrl.scmcontainermanager.StuckQuasiClosedContainers;
+                    $scope.statistics.nodes.data.container.open_without_pipeline = ctrl.scmcontainermanager.OpenContainersWithoutPipeline;
```

Comment:
Let's separate the prefix between the container lifecycle and health metrics:

- Container lifecycle metrics: `statistics.container.lifecycle`
- Health metrics: `statistics.container.health`